tests/unit/api/v1/test_status.py

```diff
@@ -13,19 +13,19 @@
 #    License for the specific language governing permissions and limitations
 #    under the License.
 
-from tests.unit import test
+import mock
+
+from tests.unit import test  # noqa
```

Comment:
"# noqa" should be removed. There was a temporarily workarond when there was an issue with tests, not everything should work fine without "# noqa"